src/app/directives/documentation.tpl.html

```diff
@@ -181,9 +181,17 @@ <h4 class="raml-console-resource-body-heading">
             <span ng-click="changeType($event, key, code)" ng-class="{ 'raml-console-is-active': $first}" class="raml-console-flag" ng-repeat="(key, value) in methodInfo.responses[code].body">{{key}}</span>
           </h4>
 
-          <div ng-if="responseInfo[code][responseInfo[code].currentType].example">
-            <span>Example:</span>
-            <pre class="raml-console-resource-pre"><code class="raml-console-hljs" hljs source="getBeatifiedExample(responseInfo[code][responseInfo[code].currentType].example)"></code></pre>
+          <div ng-if="examples">
+            <span>Examples:</span>
+            <span
+              class="raml-console-flag"
+              ng-class="{ 'raml-console-is-active': currentExample === $index }"
+              ng-repeat="example in examples"
+              ng-click="changeExample($index)">{{example.name}}</span>
+
+            <pre ng-if="currentExample === $index" class="raml-console-resource-pre" ng-repeat="example in examples">
```

Comment:
I don't think you need `ng-repeat` and `ng-if` here. Just use `examples[currentExample]`.